### PR TITLE
Several improvements to solve several issues

### PR DESCRIPTION
**Pull request by drmenzelit**
Pull Request for Issues #98 #97 #92 .

### Summary of Changes
. Remove display:grid for top and bottom positions -> improve display for mod_news on top-a
. Move rtl definition for footer to _footer.scss ([dir=rtl] & )
. Add margin to form control-group - #92 
. Add underline for links without class (e.g. in text) -> #98 
. Improve underline rules for links on menus -> #97 
. Change padding and position for mod-list on sidebar
. Improve links in mod-menu on sidebar

### Testing Instructions
Run npm ci

### CODE REVIEW

**Comment by chmst** [comment]
Thank you!  issues #97 and #98 are resolved. 

For issue #92, it is only partially as you see in the screen.

grafik

**Review by chmst — APPROVED** [review]

File: templates/cassiopeia/scss/blocks/_form.scss [hunk 1]
```diff
@@ -108,7 +108,23 @@ td .form-control {
   bottom: 100%;
 }
 
+fieldset + fieldset {
+  margin-top: $cassiopeia-grid-gutter*2;
+}
+
+fieldset:only-child {
```
Comment (hans2103) [inline templates/cassiopeia/scss/blocks/_form.scss]: where is this needed?

File: templates/cassiopeia/scss/blocks/_form.scss [hunk 2]
```diff
@@ -108,7 +108,23 @@ td .form-control {
   bottom: 100%;
 }
 
+fieldset + fieldset {
```
Comment (hans2103) [inline templates/cassiopeia/scss/blocks/_form.scss]: can I suggest to write this as follow:

```
fieldset {
  + fieldset {
    margin-top: $cassiopeia-grid-gutter*2;
  }

  > * + * {
    margin-top: $cassiopeia-grid-gutter*2;
  }
}
```

and remove the margin-top on the control-group

Comment (drmenzelit) [reply templates/cassiopeia/scss/blocks/_form.scss]: If this solve the problem of spacing between fieldsets on the forms, we can take it.
You can have a look at "site settings" and "user profile" when you login on the frontend.

Comment (hans2103) [reply templates/cassiopeia/scss/blocks/_form.scss]: discussed on Glip.

**Comment by hans2103** [comment]
> Not quite. Now the form for settings
> grafik

Created PR https://github.com/joomla/joomla-cms/pull/30737 to remove obsolete styling. When merged this part of the page looks great too.

**Comment by chmst** [comment]
This last change removes all spacing:

The image and it's select box is another issue than the spacing.

grafik

**Comment by richard67** [comment]
@drmenzelit @hans2103 Please check @chmst 's [comment above](https://github.com/joomla/cassiopeia/pull/99#issuecomment-697206046) and agree where to fix it, in this PR here or another one.

**Comment by drmenzelit** [comment]
The problems with spacing in forms should be solved with last commit.
For the form "Site settings" there is a PR from @hans2103 to core: https://github.com/joomla/joomla-cms/pull/30737

**Comment by richard67** [comment]
@chmst @hans2103 Could you repeat your tests and report back if ok? Thanks in advance.

**Comment by chmst** [comment]
Tested successfully 👍

**Review by richard67 — APPROVED** [review]

**Comment by richard67** [comment]
I have tested this PR :white_check_mark: with success.

**Comment by Quy** [comment]
No spacing.

**Comment by richard67** [comment]
@Quy Could you make a new issue for your comment?

**Pull request merged**